src/main/drivers/pwm_output.c

```diff
@@ -48,6 +48,8 @@ static uint16_t freqBeep = 0;
 static bool pwmMotorsEnabled = false;
 static bool isDshot = false;
 
+#define CHAN_TO_HALCHAN(n) (((n) - 1) * 4)
+
 static void pwmOCConfig(TIM_TypeDef *tim, uint8_t channel, uint16_t value, uint8_t output)
```

Comment:
This function is designed to mimic stdPeriph behavior, channel should be TIM_CHANNEL_x